server/ai_mediaserver.go

```diff
@@ -555,6 +556,9 @@ func (ls *LivepeerServer) cleanupLive(stream string) {
 	pub, ok := ls.LivepeerNode.LivePipelines[stream]
 	delete(ls.LivepeerNode.LivePipelines, stream)
 	ls.LivepeerNode.LiveMu.Unlock()
+	if monitor.Enabled {
+		monitor.AICurrentLiveSessions(len(ls.LivepeerNode.LivePipelines))
+	}
```

Comment:
@mjh1 @j0sh Are we sure that `cleanupLive()` is called always, even in case of errors, panics, etc?

The number of used in-use orchestrators seems high https://eu-metrics-monitoring.livepeer.live/grafana/d/be6llteqebk00b/ai-overview-livestream?orgId=1&from=1734220800000&to=now&viewPanel=6

Reply:
I'm wondering if we might be double counting somehow, the current number is 12 which is double Eric's number of 24/7 streams

Reply:
@mjh1 I think the versions with / without music are separate streams

Reply:
Yeah I was wondering that, I think the added music streams are just regular studio streams only though https://github.com/ericxtang/random-ai-stream